src/js/components/Calendar/Calendar.js

```diff
@@ -889,6 +889,10 @@ const Calendar = forwardRef(
           {daysOfWeek && renderDaysOfWeek()}
           <Keyboard
             onEnter={() => (active !== undefined ? onClick(active) : undefined)}
+            onSpace={(event) => {
```

Comment:
this is causing the lint error of 
```
 Expected an assignment or function call and instead saw an expression  no-unused-expressions
```
to fix think we can simplify to be 

```
          onSpace={(event) => {
              event.preventDefault();
              if (active !== undefined) {
                onClick(active);
              }
            }}
```

Reply:
thanks for the change fixed it

Reply:
thank you!